server/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.8-slim-buster
+
+WORKDIR /server
+
+# Copy the code of the server directory to /server of the machine
+COPY . .
+
+# install flask
+RUN pip install flask flask-cors
```

Comment:
For my new PR, I am using `pipenv`, so it can be installed. `pipenv` is basically a package manager, which will be used to install all the dependencies in the `pipfile` (kinda of package.json)

Reply:
Btw, I installed `nodemon` globally using npm so that the app is run continuously, I am not sure if we want to keep it when we run the app in a container